app/javascript/onboarding/components/ClosingSlide.jsx

```diff
@@ -8,6 +8,16 @@ class ClosingSlide extends Component {
     updateOnboarding('closing slide');
   }
 
+  whatsNextLinks() {
+    const properties = [
+      { href: '/welcome', text: 'Join the Welcome Thread', emoji: '😊' },
+      { href: '/new', text: 'Write your first DEV post', emoji: '✍️' },
+      { href: '/top/infinity', text: 'Read all-time top posts', emoji: '🤓' },
+      { href: '/settings', text: 'Customize your profile', emoji: '💅' },
+    ];
+    return properties;
+  };
```

Comment:
Wouldn't it be better to return the list instead of put it on a variable?
```js
whatsNextLinks() {
  return [
    { href: '/welcome', text: 'Join the Welcome Thread', emoji: '😊' },
    { href: '/new', text: 'Write your first DEV post', emoji: '✍️' },
    { href: '/top/infinity', text: 'Read all-time top posts', emoji: '🤓' },
    { href: '/settings', text: 'Customize your profile', emoji: '💅' },
  ];
}
```
🤔

Reply:
This is actually better. Made that way for better readability for me, but this makes more sense and uses less lines. Thanks for the review and tip, fixed it :)